tools/yaml-templates/ios-test.yml

```diff
@@ -77,17 +71,43 @@ steps:
     displayName: 'Run sample test app in background'
     workingDirectory: '$(ClientSdkProjectDirectory)'
 
-  - bash: 'sh disable_slow_animation.sh'
+  - bash: cd ~/Library/Preferences
+      defaults write com.apple.iphonesimulator SlowMotionAnimation -bool NO
     displayName: 'Disable Slow Animation'
-    workingDirectory: '$(Agent.BuildDirectory)/iOSHost/jobs/ciDevTools'
+    workingDirectory: '$(System.DefaultWorkingDirectory)'
 
-  - bash: 'sh parallel_ui_testing_for_publishing.sh'
```

Comment:
Are these two sh files being used by ios hub sdk? If they are used by two places, one potential issue is for any fixes/improvement done in the script, teamsjssdk won't be able to pick it up and we have to remember to update here manually

Reply:
I separated them from Teams-JS library and iOS host last time. Also, to reduce dependency, I am trying to get rid of these scripts so that we won't accidentally edit it and then break the pipelines in Teams-JS library side.